### ISSUE TITLE
Reduce annoyance of early power bomb seeds

### ISSUE THREAD

**Issue by Miepee** [MetroidAdvRandomizerSystem/MARS-Fusion#99]
When starting with power bombs, theres a very high chance that you might need to farm off of enemies, or constantly go back and forth between recharge stations.
Bumping the chance for a red x would be helpful to be either 1/256 or 1/128.

**Comment by biosp4rk** [comment]
What about for enemies that already have a higher rate to drop red X? Or are you just saying the minimum should be 1/256?

**Comment by JeffGainsNGames** [comment]
I believe it's the latter.

**Comment by Miepee** [comment]
Yeah if enemies already have a higher chance (e.g. blue zoros) there's no need to nerf that.

**Comment by zarakava** [comment]
I've started taking a look at this to get my feet wet in Assembly again, and I've got a question:

Do we want to treat the fix to this issue simply as adjusting the drop rates for all enemies to have 1/256 odds, or do we want to look at doing something different for the actual drop determination function - we could do something like guarantee the 50th kill gives a red X.

**Comment by Miepee** [comment]
I'm personally mostly indifferent as long as 1) Gameplay isn't affected *too* much 2) the goal of reduced annoyance is achieved

**Comment by jasonbbb711** [comment]
Not sure if the implementation of this would be a pain, but perhaps extra spawns for blue Zorros or pink Zeelas in some of the drier areas could work? Would be less annoying to grab a red X from one of those than farming dozens of low level enemies for a rare drop.

**Comment by Miepee** [comment]
Having adjusted drop rates isn't anything you have to remember, you kill a few enemies until you get a drop.

Adjusted enemies needs to be remembered on where they are, then potentially your movement needs to be adjusted to not hit them, and then their effect is really hard to gauge because they could be either very powerful or very useless depending on the settings.
